### ISSUE TITLE
Error requesting initial Data for Sensor-Matching: null

### ISSUE THREAD

**Issue by octaviz** [dehsgr/homebridge-mobilealerts#19]
Hello, since the last update 1.4.0 I get an error message, the sensors work and update correctly, but I get the message all the time, App Home & Eve reciving data without errors.

The Error:
[4/19/2021, 12:59:45 PM] [MobileAlerts] There was an Error requesting initial Data for Sensor-Matching: null

Thanks!

**Comment by dehsgr** [comment]
@octaviz could you set the log level to verbose with HTML output according to config sample? Further I would like you to send the log to dehsgr at sign outlook dot sign com.

**Comment by dehsgr** [comment]
@octaviz alternatively you should send me you MA ID.

**Comment by octaviz** [comment]
> 
> 
> @octaviz could you set the log level to verbose with HTML output according to config sample? Further I would like you to send the log to dehsgr at sign outlook dot sign com.

Ok!, Here you have the Vervose / HTML rename to HTML log

**Comment by octaviz** [comment]
> 
> 
> @octaviz alternatively you should send me you MA ID.

I don't know where to see the MA id code, these are the sensors I have installed: 

https://www.tfa-dostmann.de/en/product/digital-thermo-hygrometer-cosy-radar-30-5043/
https://www.tfa-dostmann.de/en/product/temperature-humidity-transmitter-with-water-detector-weatherhub-30-3305/
https://www.tfa-dostmann.de/en/product/temperature-transmitter-weatherhub-30-3300/

Thanks

**Comment by dehsgr** [comment]
@octaviz no problem. MA ID is your iPhone ID. I got it from attached HTML (I deleted the link from your comment now to save your data). 

I'll recently publish an enhancement fix that will clarify HTTP status code. There is currently some maintenance at mobile alerts site and server seems to respond with code 429 (too many requests) and error "The custom error module does not recognize this error". Nothing I can do here.

Did you set a custom polling interval? If yes I would like you to remove it from your config. This might cause the issue.
